CUDADataFormats/BeamSpot/interface/BeamSpotCUDA.h

```diff
@@ -0,0 +1,44 @@
+#ifndef CUDADataFormats_BeamSpot_interface_BeamSpotCUDA_h
+#define CUDADataFormats_BeamSpot_interface_BeamSpotCUDA_h
+
+#include <cuda_runtime.h>
+
+#include "HeterogeneousCore/CUDAUtilities/interface/device_unique_ptr.h"
+
+class BeamSpotCUDA {
+public:
+  // align to the CUDA L1 cache line size
+  struct alignas(128) Data {
+    float x, y, z;  // position
+    // TODO: add covariance matrix
```

Comment:
Are there plans now to add the cov matrix here?

Reply:
I hasn't been added because the GPU clients don't seem to need it.
I guess it will be added if a new client appears that needs it, or if any other reason comes up.